carla/models/catalog/catalog.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+
+from ..api import MLModel
+from .load_model import load_model
+
+
+class MLModelCatalog(MLModel):
+    def __init__(
+        self, data_name, model_type, ext="h5", cache=True, models_home=None, **kws
```

Comment:
Should we abstract away the `extention` and rather pass the backend directly and derive the extension from it? So the user of this api doesn't need to know what exactly yhe extension of pytorch and tensorflow models are and we save an if/else condition

Reply:
Right, with backend it would be more understandable to the user. Changes will be in the next commit.